playbooks/roles/database/tasks/main.yml

```diff
@@ -79,17 +79,32 @@
       when: ansible_os_family|lower == "suse"
   when: database == "mariadb" and database_options.add_repo
 
+- block:
+    - name: install the PostgreSQL repository (yum)
+      yum_repository:
+        name: "{{ postgres_repo_name }}"
+        description: "PostgreSQL"
+        baseurl: "{{ postgres_repo_url }}"
+        state: present
+        gpgcheck: yes
+        gpgkey: "{{ postgres_repo_key }}"
+      when: ansible_os_family|lower == "redhat"
+  when: database == "postgres" and (postgres_repo_url|default(''))!=''
+
 - name: Install {{ database }} database packages
   package:
     name: "{{ item }}"
     update_cache: yes
     state: present
+#    enablerepo: "{{ postgres_enablerepo | default(omit) }}"
```

Comment:
I forgot to remove that comment .. but since the new feature https://github.com/hortonworks/ansible-hortonworks/commit/de71ccf4918b6e17df4bb297c6b50f8a5b3603cf 
enablerepo should probably also be used for postgres!

Reply:
comment removed and added commit
> external postgres repository only used if (new) option add_repo enabled